packages/clients/generic/API.md

```diff
@@ -6,6 +6,8 @@ For all additional details, check the [full documentation](https://dataport.gith
 
 For the development test deployments and example configurations, [see here](https://dataport.github.io/polar#plugin-gallery).
 
+For a minimum working example, [see here](https://github.com/dopenguin/polar-fossgis-2024).
```

Comment:
I would prefer having an MWE in the POLAR repo itself. An external example will most likely not be kept up to date, and there is a SPOF, because it is in a personal repo.

Reply:
An external example yields the benefit for potential users that they can just check out the repository and get started.

The example is supposed to be kept up to date with the current state of `@polar/client-generic` as it is also linked for potential users that listened to the FOSSGIS 2024 talk.

We could think of moving the repository to the dataport company, but I figured that that leads to a lot of overhead.

Reply:
Agreed. Moving to the dataport organization is IMO the way to go, but we can postpone that.